react/components/BuyButton/assemblyUtils.js

```diff
@@ -1,12 +1,26 @@
-import { filter, isEmpty } from 'ramda'
+import { pathOr } from 'ramda'
 
-const filterAssembliesWithItem = filter(items => items.length > 0)
+export const sumAssembliesPrice = assemblyOptions => {
+  const cleanAssemblies = assemblyOptions || {}
+  const assembliesGroupItems = Object.values(cleanAssemblies)
```

Comment:
Object.values nao funciona no IE. ve se nao tem um equivalente no ramda

Reply:
alias pode ser que esteja vindo no polyfill.io 🤔 mas acho que ja que ja esta sendo usado o ramda, daria pra pegar uma funçao similar de lá de uma vez

Reply:
No worries. Object.values is polyfilled

![image](https://user-images.githubusercontent.com/284515/61892274-cd7e6c80-aee1-11e9-8955-1be2962ea2a0.png)